### ISSUE TITLE
[bug] aiohttp.post is broken

### ISSUE THREAD

**Issue by carlcarl** [aio-libs/aiohttp#461]
python version: 3.4.3
aiohttp==0.17.1

Hi, the `post` method is broken because it can't include `data` parameter, here is my test code and result:

```
import asyncio
import aiohttp

url = 'http://127.0.0.1:1234'
data = {'a': '11111'}

loop = asyncio.get_event_loop()

loop.run_until_complete(asyncio.wait([aiohttp.post(url, data=data)]))
```

Result(using `nc -l 1234`):

```
POST / HTTP/1.1
ACCEPT: */*
ACCEPT-ENCODING: gzip, deflate
HOST: 127.0.0.1:1234
CONTENT-LENGTH: 0
CONTENT-TYPE: application/octet-stream
CONNECTION: keep-alive
USER-AGENT: Python/3.4 aiohttp/0.17.1
```

if I use `request` instead of `post`, the result is OK:

```
POST / HTTP/1.1
ACCEPT: */*
ACCEPT-ENCODING: gzip, deflate
HOST: 127.0.0.1:1234
CONTENT-TYPE: application/x-www-form-urlencoded
CONTENT-LENGTH: 7
CONNECTION: keep-alive
USER-AGENT: Python/3.4 aiohttp/0.17.1

a=11111
```

**Comment by kxepal** [comment]
I guess `put`, `delete` and `patch` are [broken](https://github.com/KeepSafe/aiohttp/blob/master/aiohttp/client.py#L490-L511) as well. `data` argument eventually forgot to be passed forward.

**Comment by asvetlov** [comment]
Fixed. Will be published as part of 0.17.2
